extensions/arc/src/ui/dashboards/postgres/postgresComputeAndStoragePage.ts

```diff
@@ -12,6 +12,18 @@ import { DashboardPage } from '../../components/dashboardPage';
 import { PostgresModel } from '../../../models/postgresModel';
 import { convertToGibibyteString } from '../../../common/utils';
 
+export type ConfigurationSpecModel = {
+	workers?: number,
+	workerCoresRequest?: string | undefined,
```

Comment:
FYI - the undefined isn't needed at this point since it's already an optional property